doc/man3/OSSL_ENCODER_CTX.pod

```diff
@@ -144,18 +155,26 @@ or NULL to indicate that an error has occured.
 
 These utility functions may be used by a constructor:
 
-OSSL_ENCODER_INSTANCE_encoder() can be used to get the encoder method from a
-encoder instance I<encoder_inst>.
+OSSL_ENCODER_INSTANCE_get_encoder() can be used to get the encoder method
```

Comment:
encoder method is not described here..

Reply:
Rephrased

Reply:
Note that encoder method is all over the place in here..